pkg/admin/server/envoy.go

```diff
@@ -0,0 +1,51 @@
+package server
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+
+	"github.com/golang/protobuf/jsonpb"
+	"mosn.io/mosn/pkg/admin/store"
+	"mosn.io/mosn/pkg/log"
+	"mosn.io/mosn/pkg/xds/conv"
+	envoyControlPlaneAPI "github.com/envoyproxy/go-control-plane/envoy/admin/v2alpha"
+
+)
+
+func statsForEnvoy(w http.ResponseWriter, r *http.Request) {
```

Comment:
should rename `statsForEnvoy`  to `statsForIstio` or `statsForMosn` ?

Reply:
`statsForIstio` seems better